server/routerlicious/packages/protocol-base/.eslintrc.cjs

```diff
@@ -20,5 +20,8 @@ module.exports = {
 
 		// TODO: remove usages of deprecated APIs and remove this override
 		"import/no-deprecated": "warn",
+		// This package uses node's events APIs.
+		// This should probably be reconsidered, but until then we will leave an exception for it here.
+		"import/no-nodejs-modules": ["error"],
```

Comment:
This is already set like this in the base configs (including minimal-deprecated referenced above) so we don't need to specify it here.